### ISSUE TITLE
Add /help/contact to website

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#34]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is to expand the `/help/contact` URL for activist.org. The Vue page in question is [pages/help/contact.vue](https://github.com/activist-org/activist/blob/main/frontend/pages/help/contact.vue).

The page will be a contact form that will email a common email address for the organization. This email has yet to be set up, but likely will be `team@activist.org`. We'll also need to include some descriptions of who we'd like to hear from and thank them for contacting us.

### Contribution

I'll be working on this as we continue v0.0.1 development. Help and ideas are welcome!

**Comment by andrewtavis** [comment]
The email address mentioned above has been made, so we're ready to implement this :)

**Comment by 5ud03r5** [comment]
Hey, can you assign me to this one please?

**Comment by andrewtavis** [comment]
By all means, @5ud03r5! I'm working on the edits to your work in #62 right now 😊 Thanks for your interest in the project :) :)

**Comment by andrewtavis** [comment]
@5ud03r5, see the comments that I made in #62 for some directions on how best to work on this one. I think the only big thing it'd be great for you to look out for is that Tailwind style changes should be triggered after the breakpoint as I haven't been using `max-breakpoint` much in the code till now :) Breakpoints conceptually to me are parts where changes occur after them, which I hope is ok with you 😊

Let me know if you need any support or have any questions!

**Comment by andrewtavis** [comment]
Yes I definitely can, @5ud03r5 😊 Sorry, this is a known issue in #55. I made a change so that we can get hot reloading working again on my end, but then it seems to break Docker for others 🤔 I'll do an update now that should fix it :)

**Comment by andrewtavis** [comment]
@5ud03r5, af10033 should fix this issue for you as I reverted a change I made 😊 Let me know if the problem persists 🙃

**Comment by andrewtavis** [comment]
ea87f1a adds the text and does some formatting on [/help/contact/](https://activist.org/en/help/contact/), @5ud03r5 :) I need to do some research to better understand what all I need to change to get the email working as you directed in your [comment in the PR](https://github.com/activist-org/activist/pull/78#issuecomment-1480994962), but it should be good to go soon 😊

**Comment by andrewtavis** [comment]
@to-sta: 29cda4a did a rework of the layout to match the rest of the docs 😊 See [activist.org/help/contact](https://activist.org/help/contact/). Let's figure out the message send and call it done ✅

**Comment by to-sta** [comment]
@andrewtavis while looking at the contact form, i was wondering why there is no subject line. Any particular reason for this?

**Comment by andrewtavis** [comment]
Good call, @to-sta :) We should definitely include that as well!

**Comment by to-sta** [comment]
@andrewtavis only thing left on this is to get a working smtp server. I have tested it with a free smtp server everything worked well 😃

**Comment by Stegen54** [comment]
Is this issue still available,I will like to work on it. @andrewtavis

**Comment by andrewtavis** [comment]
Is still something we're trying to solve, @Stegen54 :) Big thing is that we've really wanted to get this up and running with a [Proton Bridge](https://proton.me/mail/bridge) to make sure that the traffic is routed through there, but thinking about this, maybe it makes sense for now to implement this with [nuxt-mail](https://nuxt.com/modules/nuxt-mail) that we have in the dependencies and put a note that's saying that the means of transfer isn't the most secure. Happy to discuss this further and for a PR!

**Comment by 0priyamdey** [comment]
Is it still open, sir can you assign it to me.

**Comment by andrewtavis** [comment]
Hey @0priyamdey 👋 This really isn't something we're able to assign to the broader community. We'll get to the current high priority tasks and then the core team will focus on this. Thanks for your interest!

**Comment by NajibSampleProject** [comment]
Hi Andrew, I love to work on this issues, can you please assign me to this? Much appreciated.

**Comment by NajibSampleProject** [comment]
Hi @andrewtavis, I love to work on this issues as well, can you please assign me to this as contributors too? Much appreciated.

**Comment by andrewtavis** [comment]
Hey @NajibSampleProject 👋 This is one of the longest standing issues that we have, as you can see. Maybe you can do some research on Proton bridges for sending emails and we can look into deploying something based on what you find?

Thanks for your interest in supporting here! Feel free to write in the issue with what you find :)

**Comment by Nishthajain7** [comment]
@5ud03r5 are you still working on it ? If not I'd like to take this up

**Comment by andrewtavis** [comment]
Thanks for your offer to help, @Nishthajain7 :) If you wanted to look into the similar research I prompted @NajibSampleProject in [my message above](https://github.com/activist-org/activist/issues/34#issuecomment-2868902731), then that'd be great! I'll write back to you in the other issue you wrote in soon as well :)

**Comment by NajibSampleProject** [comment]
Hey @Nishthajain7 , in relation to your question, maybe you ask @andrewtavis to add you if you're interested -- I am currenlty working on this issues (#34) DOING SOME RESEARCH rn -- need to investigage all the components before come to conclusion.

**Comment by andrewtavis** [comment]
Sounds good to me, @NajibSampleProject :) @Nishthajain7 can look into #716, and please let us know when you have an idea here 😊

**Comment by NajibSampleProject** [comment]
Hey @andrewtavis I have started working and researching on this issue --Found some solutions--, I will shortly start sharing my finding in relation to Proton-Bridge and a draft solutions. Cheers

**Comment by andrewtavis** [comment]
Really great to hear this, @NajibSampleProject :) Thanks so much for your efforts! 😊

**Comment by ydureix** [comment]
Hi! I'm new to open source and noticed this issue, which looks like a great first opportunity. I have experience with fullstack development and would love to take a crack at it. Please let me know if it’s still available? Thanks!

**Comment by ydureix** [comment]
Hi @andrewtavis ! I'm new to open source and noticed this issue, which looks like a great first opportunity. I have experience with fullstack development and would love to take a crack at it. Please let me know if it’s still available? Thanks!

**Comment by andrewtavis** [comment]
Hey @ydureix 👋 It'd be great to get support here, but also this one is a bit of a minefield at this point. Big thing is that we want to be sending the final message via Proton so that the email doesn't go through a non-trusted source. As we understand it we can do this via Proton bridges, but there just hasn't been capacity to look into setting it up properly. If you would likely to check it out and get back to us, then you'd be very welcome to! You'd be welcome to switch to another issue though as well :) Don't stress on getting to this one 😊

**Comment by ydureix** [comment]
@andrewtavis I'll give it a shot 😃

**Comment by andrewtavis** [comment]
Nice @ydureix! Please let us know if you need some support or are getting a bit stuck :)

**Comment by manjaycoder** [comment]
can you assign me these issue

**Comment by andrewtavis** [comment]
Blocking this for now as we need to look into this internally in the team. @manjaycoder, I mistyped in the last message. The file is `CONTRIBUTING.md`. I'd ask that you take some time to familiarize yourself with it and then ask to be assigned an issue.

**Comment by mattburnett-repo** [comment]
Hey @andrewtavis ! I'm self-assigning because this is related to a dependency mentioned in Issue #2051.

Short version:
Issue 2051 has a requirement to send notification emails when a triggering event occurs. In order to do that, I will need to stand up some infrastructure for sending out emails. Other contact media can be added later, but for now, Issue 2051 will need basic email functionality in order to work as designed.

I think it likely, that the block faced by this Issue will be resolved by working through the implementation of 2051.

**Comment by andrewtavis** [comment]
Thanks for looking into this, @mattburnett-repo!

**Comment by andrewtavis** [comment]
@mattburnett-repo, talking about this with @to-sta and @sh-ran right now. One note is that we're thinking that a separate service won't be needed for this as it's so integrated into the backend itself in terms of the usage of sign up and filescan for the content application within the backend. @to-sta's suggestion is to look into using Django tasks for this, which will allow us to trigger this after certain responses.

**Comment by mattburnett-repo** [comment]
> [@mattburnett-repo](https://github.com/mattburnett-repo), talking about this with [@to-sta](https://github.com/to-sta) and [@sh-ran](https://github.com/sh-ran) right now. One note is that we're thinking that a separate service won't be needed for this as it's so integrated into the backend itself in terms of the usage of sign up and filescan for the content application within the backend. [@to-sta](https://github.com/to-sta)'s suggestion is to look into using Django tasks for this, which will allow us to trigger this after certain responses.

Agreed about the separate service; I think any discussion about implementing notification as a service came up in the context of discussing the filescan thing. How we implement notifications isn't as relevant in this thread, but rather that we simply implement notification as part of the follow-on tasks for the filescan implementation. Implementing notification for filescan just points the way for implementing it across the entire backend, thereby addressing the needs of this issue.

TL;DR: You're right. All that really matters is that notification functionality is implemented. I'm not committed to a specific way of doing this; whatever works well in the context of the existing architecture works for me.

**Comment by MayurK-cmd** [comment]
Hey Can i work on this?

**Comment by andrewtavis** [comment]
We're basically done with this at the moment, @MayurK-cmd. Maintainers will do a call in the coming weeks just to verify that all's working as expected.
